export default function Test() {
    throw new Error("에러발생");
}
